src/tools/stereoCalib/src/stereoCalibThread.cpp

```diff
@@ -219,7 +244,7 @@ void stereoCalibThread::stereoCalibRun()
                     stereoCalibration(imageListLR, this->boardWidth,this->boardHeight,this->squareSize);
 
                     yInfo(" Saving Calibration Results... \n");
-                    updateIntrinsics(Left.cols,Left.rows,Kright.at<double>(0,0),Kright.at<double>(1,1),Kright.at<double>(0,2),Kright.at<double>(1,2),DistR.at<double>(0,0),DistR.at<double>(0,1),DistR.at<double>(0,2),DistR.at<double>(0,3),"CAMERA_CALIBRATION_RIGHT");
+                    updateIntrinsics(Right.cols,Right.rows,Kright.at<double>(0,0),Kright.at<double>(1,1),Kright.at<double>(0,2),Kright.at<double>(1,2),DistR.at<double>(0,0),DistR.at<double>(0,1),DistR.at<double>(0,2),DistR.at<double>(0,3),"CAMERA_CALIBRATION_RIGHT");
```

Comment:
Woh.. changing from `left` to `right` seems a big modification. From the PR review dashboard, it's almost impossible to judge it. Could you add here some hints on that?

Reply:
That function call is actually referred to the intrinsic parameter update for the Right camera. It did not make the difference so far, because the two images have also been assumed to have the same size (cols and rows). However in this pull request we are not making this assumption anymore and allowing the two images to have different sizes, as long as the intrinsic parameters are calibrated separately beforehand (throwing an error otherwise).

Reply:
Ok then! It's been actually a bug fix too 👍